pages/landing/patcher/pricing/index.html

```diff
@@ -1,72 +1,81 @@
 ---
 layout: landing
 type: landing
-slug: index-page
+slug: pricing
 
-title: Pricing
+title: Pick a plan
 excerpt: Configure the plan just right for your team.
```

Comment:
```suggestion
excerpt: Pick the plan just right for your team.
```
I suggest we change this heading as I feel our current pricing structure doesn't include optional add-ons, therefore to me it feels we are not 'configuring' a plan, but rather selecting one.